types/office-js/index.d.ts

```diff
@@ -4440,7 +4440,7 @@ declare namespace Office {
          *   </tr>
          *   <tr>
          *     <td><code>Office.CoercionType.SlideRange</code></td>
-         *     <td>PowerPoint on the web and Windows</td>
+         *     <td>PowerPoint on the web and on Windows</td>
```

Comment:
Consider formatting the applications in this table to look like the equivalent in office-js-preview
```suggestion
         *     <td>- PowerPoint on the web and on Windows</td>
```

Reply:
Done!